data/mods/Magiclysm/items/cast_spell_items.json

```diff
@@ -394,5 +394,37 @@
     "use_action": { "type": "cast_spell", "spell_id": "twisted_restore_improved", "no_fail": true, "level": 1 },
     "type": "COMESTIBLE",
     "copy-from": "twisted_restore_potion"
+  },
+  {
+    "id": "forge_warp_scroll",
+    "name": { "str": "Forge of Wonders warp scroll" },
+    "description": "This scroll is imbued with a magical spell that, when activated, will transport you to the Forge of Wonders.  Good for one round-trip.",
+    "looks_like": "spell_scroll",
+    "use_action": { "type": "cast_spell", "spell_id": "forge_teleport", "no_fail": true, "level": 1 },
+    "type": "GENERIC",
+    "category": "other",
+    "weight": "475 g",
+    "volume": "500 ml",
+    "//": "Price should be 300 but Velzain charges almost double. If this gets moved to another vender bump up price to 300.",
```

Comment:
I noticed a typo in this line but I'm not going to make it redo tests for a non player facing comment.